syntax = "proto3";

package bayesmix;

enum HierarchyId {
    UNKNOWN_HIERARCHY = 0;
    NNIG = 1;
    NNW = 2;
    LinRegUni = 3;
}
